### ISSUE TITLE
"I'm definitely going"

### ISSUE THREAD

**Issue by isaacs** [mikeal/gatherbugs#47]
It'd be nice to have a way to mark that you're definitely going to a gather, and then see who has marked it thus.  Even though rsvp's are always partly a lie, they're rarely _completely_ a lie, so in aggregate, it'd be great info.  If you could then un-mark yourself as going toggle-style, that'd also make the data more reliable.

For extra points: if I've said that I'm definitely going, give me a push notification an hour before the event.

**Comment by mikeal** [comment]
we've been talking about adding a workflow around the foursquare checkin so it's not just a "checkin" it's an "i'm here" and anyone who is following it but close would get a notification about it.

saying you're there when you're there would mean a lot more than RSVP, which is usually a lie and even at it's best is an optimistic yet shallow commitment :)

**Comment by isaacs** [comment]
Right, but if I post a gather, and no one says they're coming, I might not go.

If I see a gather and my ex says they'll be there, I might want to avoid it.

If someone I don't like much posts a gather, and someone I like will be there, I might go even though it doesn't look too exciting.

Saying "i'll be there", even if it's untrue a lot of the time, is true even more of the time, and is part of the conversation that makes "people in places doing things" more engaging.

**Comment by isaacs** [comment]
That being said, yes, an "i'm here" mechanism would also be great :)

**Comment by max-mapper** [comment]
we should figure out the most lightweight way to let people say both 'i am on the way' and 'i am here'

**Comment by apage43** [comment]
If they've marked themselves as going to a gather, perhaps with the notification/reminder also ask them the confirm that they are in fact attending. At that point periodically poll their location, considering them "on the way" until they get near the event, then mark them as at the event once they arrive. Perhaps also notify them that this has happened, but I think that it'd be safe to do that without confirmation.

**Comment by apage43** [comment]
Also, it might be worth just doing periodic polling for everyone, and if they happen to be near an event that would be in their feed, even if they didn't RSVP, ask them to check in.

**Comment by mikeal** [comment]
that's an interesting idea: we could ad reminders and surface the anonymized number of reminders people have set for that event.

**Comment by max-mapper** [comment]
the geoloqi SDK would be useful here, i'm planning on diving into it in the future
